cores/esp32/esp32-hal-timer.c

```diff
@@ -194,7 +187,7 @@ static void _on_apb_change(void * arg, apb_change_ev_t ev_type, uint32_t old_apb
 hw_timer_t * timerBegin(uint8_t num, uint16_t divider, bool countUp){
     if(num >= NUM_OF_TIMERS)
     {
-        log_e("Timer dont have that timer number.");
+        log_e("'num' exceeds available number of Timers.");
```

Comment:
the `'num'` is not printing the num passed to the function.
Please fix it to something like this:
`log_e("Timer number %u exceeds available number of Timers.",num);`

Reply:
Fixed.